docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
Wow, OK, my bad, didn't realize that description was plain text and not parsed HTML.  Thought that the pattern wouldn't handle the "<code>" blocks.  👏